### PR TITLE
authorize: round timestamp

### PR DESCRIPTION
**Pull request by wasaga**
## Summary

round timestamp for 'exp' and 'iat' claim that some SDKs expect to be whole unix seconds.

## Related issues

#2248 

## Checklist

- [x] reference any related issues
- [ ] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

**Review by calebdoxsey — APPROVED** [review]

File: authorize/evaluator/opa/policy/headers.rego [hunk 1]
```diff
@@ -142,7 +142,7 @@ base_jwt_claims := [
 	["iss", jwt_payload_iss],
 	["aud", jwt_payload_aud],
 	["jti", jwt_payload_jti],
-	["exp", jwt_payload_exp],
+	["exp", round(jwt_payload_exp)],
 	["iat", jwt_payload_iat],
```
Comment (desimone) [inline authorize/evaluator/opa/policy/headers.rego:146]: and iat

File: authorize/evaluator/opa/policy/headers.rego [hunk 2]
```diff
@@ -224,10 +224,10 @@ identity_headers := {key: values |
 
 	some i
 	[key, v1] := h[i]
-	values := [ v2 |
-	    some j
-	    [k2, v2] := h[j]
-	    key == k2
+	values := [v2 |
+		some j
```
Comment (desimone) [inline authorize/evaluator/opa/policy/headers.rego:228]: unrelated changes

**Review by desimone — CHANGES_REQUESTED** [review]
Could you add a unit test to confirm?

File: authorize/evaluator/opa/policy/headers.rego (hunk 2 above)
Comment (wasaga) [reply authorize/evaluator/opa/policy/headers.rego:228]: the original block was a mix of tabs & spaces formatting. We customary fix formatting issues part of other PRs.

**Review by desimone — APPROVED** [review]
